recipe/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: rust_matchspec
+  version: 0.1.0
+
+source:
+  path: ../
+
+build:
+  script: $PYTHON -m pip install --no-deps .
+
+requirements:
+  host:
+    - python >=3.7
```

Comment:
You can remove the python pinnings and instead add `skip: true  #[py<37]` inside the build section.